Plugins/GitHub3/GitHubHostedRemote.cs

```diff
@@ -52,5 +52,10 @@ public IHostedRepository GetHostedRepository()
         public string Data => Owner + "/" + RemoteRepositoryName;
         public string DisplayData => Data;
         public bool IsOwnedByMe => GitHubLoginInfo.Username == Owner;
+
+        public string GetBlameUrl(string commitHash, string fileName, int lineIndex)
+        {
+            return $"https://github.com/{Data}/blame/{commitHash}/{fileName}#L{lineIndex}";
```

Comment:
What about gh enterprise?

Reply:
Done something (even if there is no github enterprise support in GE at the moment)